### PR TITLE
Preferring VSCode and clarifying when not to use heroku open

### PR DESCRIPTION
**Pull request by ChadKreutzer**

### CODE REVIEW

**Comment by ChadKreutzer** [comment]
I do have a question. Isn't it extra steps to close and reopen the editor? Why **not** just have them open the app's folder in VScode, since that will be more akin to how they will actually be working later on?

**Comment by I3uckwheat** [comment]
@chadkreutzer I do believe it would be easier for then to just run `code .` to open the directory. This'll also teach them how to do it in the future.

**Comment by ChadKreutzer** [comment]
@I3uckwheat That is absolutely fine. So howabout having them do that, then just opening the Gemfile and then the config/routes.rb from inside VSCode? rather than opening it, editing the Gemfile, then closing it and then reopening it to edit the route then re-closing it.

**Comment by I3uckwheat** [comment]
@ChadKreutzer The directions are currently close to each other

Your First Rails app
---

Step 4.5.1: Configuring the Gemfile
To properly deploy a rails application, we will have to change some configuration in its files.

...

Then we will use the text editor you chose to modify our Gemfile. Open the the Gemfile in your text editor of choice.

Delete the line that says:

...

---

> Then we will use the text editor you chose to modify our Gemfile. Open the the Gemfile in your text editor of choice.

That could say:

Then we will use VSCode to open the project directory and edit the Gemfile, type `code .` to open the current directory

---

Then somewhere it could state: "Do not close your editor just yet! Continue down to the next section to edit another file"

`Step 4.5.2: Installing the Bundle` and `Step 4.5.3: Configuring the root route` could probably be swapped so they're together and the student can edit them one after another, *then* close the editor.

**Comment by I3uckwheat** [comment]
@ChadKreutzer One more thing to note: Unless the user sets up `code` in their path, `code .` does not work in MacOS, they would have to use `open .` or open vscode and

> Command + Shift + P in visual studio code then start typing Shell Command: Install 'code' command in PATH and then run that so you can open VSC from your terminal

as per @105ron

**Comment by ChadKreutzer** [comment]
@I3uckwheat Okay. I think that should rough in everything we discussed. gimme some feedback, yo!

File: web_development_101/installfest_text_editors.md [hunk 1]
```diff
@@ -39,6 +39,18 @@ VSCode is a newer editor created by Microsoft, and is completely free. It has ou
...
+
+Okay, so go install VSCode if you are going to. The link is right up there. Alright, if you are on Linux or Windows, you are good to go. If you are on MacOS, we are going to have you jump through one more hoop --trust me, it's for your own good:
+
+##### Enable code command for MacOS
+
+On Windows and Linux, you can open VSCode from the command line by typing `code` alone. And you can open folders or files by adding the name of the location after it: `code MyCoolProject`. MacOS can do it too, but you gotta set it up yourself.
+
+After installing VSCode launch it whichever way your are comfortable. Once it's running open the Command palette by typing `Ctrl+Shift+P`. In the little dialog that appears type `shell command`. One of the choices that appears will be `Shell Command: Install 'code' command in PATH`. I bet you know where this is going. Select that option and restart the terminal if you have it open.
+
+Now You with your Mac are almost as cool as your friends with Linux and Windows.
```
Comment (I3uckwheat) [inline web_development_101/installfest_text_editors.md]: Nice touch ;)

**Pull request merged**